deltachat-contact-tools/src/lib.rs

```diff
@@ -130,7 +131,47 @@ pub fn parse_vcard(vcard: &str) -> Vec<VcardContact> {
             // so this line's property is actually something else
             return None;
         }
-        Some(value)
+        if let Some(p) = remove_prefix(params, ";") {
+            params = p;
+        }
+        if let Some(p) = remove_prefix(params, "PREF=1") {
+            params = p;
+        }
+        Some((params, value))
+    }
+    fn base64_key(line: &str) -> Option<&str> {
+        let (params, value) = vcard_property(line, "key")?;
+        if params.eq_ignore_ascii_case("PGP;ENCODING=BASE64")
+            || params.eq_ignore_ascii_case("TYPE=PGP;ENCODING=b")
+        {
+            return Some(value);
+        }
+        if let Some(value) = remove_prefix(value, "data:application/pgp-keys;base64,")
+            .or_else(|| remove_prefix(value, r"data:application/pgp-keys;base64\,"))
```

Comment:
The \ appeared only in a contact which was first 
1) exported from Delta Chat to Proton, 
2) and then exported from Proton to Delta Chat.

I think it only remained in the vCard because the couldn't be imported in Proton, so Proton stored the value unmodified (except for the \) in case some other application might need it later. As soon as Proton can import public keys from our vCards probably it should be fine. But then again, caring for this case doesn't hurt.